src/load-test/prod/networking.tf

```
@@ -0,0 +1,33 @@
+resource "azurerm_virtual_network" "weu_load_test" {
+  name                = "${local.project}-load-test-vnet-01"
+  resource_group_name = azurerm_resource_group.load_test.name
+  location            = azurerm_resource_group.load_test.location
+  address_space       = ["10.40.0.0/22"]
+
+  tags = local.tags
+}
+
```

```suggestion

resource "azurerm_private_dns_zone_virtual_network_link" "azurewebsites_private_vnet_common" {
  name = azurerm_virtual_network.weu_load_test.name

  virtual_network_id    = azurerm_virtual_network.weu_load_test.id
  resource_group_name   = data.azurerm_private_dns_zone.privatelink_azurewebsites.resource_group_name
  private_dns_zone_name = data.azurerm_private_dns_zone.privatelink_azurewebsites.name

  registration_enabled = false

  tags = local.tags
}

```
I may be wrong, but DNS resolution of private dns records don't work over VNet peering if the private DNS zone is not linked to both VNets

---

That's correct! I've updated the code